src/App.js

```diff
@@ -518,13 +518,13 @@ const BlankScreen = () => null;
  */
 const RootStack = () => {
   const dispatch = useDispatch();
-  const [isLoaded, setIsLoaded] = useState(false);
+  const [appStatus, setAppStatus] = useState('initializing');
```

Comment:
💬 When the 'initializing' status is used?

Reply:
Previously there were only two states: `isLoaded: false` and `isLoaded: true`.

The problem was that the screen rendering was happening before the storage could answer the promise with the correct information. So, for a brief moment the rendering was incorrect and then the screen flickered to the correct one.

The state `appStatus: initializing` ensures no render will occur until the actual state of the wallet is fetched: either loaded or not.

As this only takes a second or two, I didn't bother adding a `loading` animation, but just a blank screen as was the behavior previously of our implementation on React Navigation 4.